### ISSUE TITLE
An illegal reflective access operation has occurred

### ISSUE THREAD

**Issue by javydreamercsw** [retest/recheck#340]
Illegal reflective access by de.retest.recheck.ui.image.ImageUtils (file:/Users/javierortiz/.m2/repository/de/retest/recheck/1.3.0/recheck-1.3.0.jar) to field sun.awt.CGraphicsDevice.scale

**Comment by javydreamercsw** [comment]
Still valid on 1.5.1:
```
# Created at 2019-09-27T14:45:26.774
WARNING: An illegal reflective access operation has occurred

# Created at 2019-09-27T14:45:26.774
WARNING: Illegal reflective access by de.retest.recheck.ui.image.ImageUtils (file:/Users/javierortiz/.m2/repository/de/retest/recheck/1.5.0/recheck-1.5.0.jar) to field sun.awt.CGraphicsDevice.scale

# Created at 2019-09-27T14:45:26.775
WARNING: Please consider reporting this to the maintainers of de.retest.recheck.ui.image.ImageUtils

# Created at 2019-09-27T14:45:26.775
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations

# Created at 2019-09-27T14:45:26.775
WARNING: All illegal access operations will be denied in a future release
```

**Comment by beatngu13** [comment]
There's a [workaround for Java 9+](https://bugs.openjdk.java.net/browse/JDK-8172962), but we have to stay on 8 and ensure forward compatability. I'm afraid we have to check the Java version at runtime and then work with reflection or do [something like this](https://stackoverflow.com/a/53538597) to make the warning disappear.

**Comment by javydreamercsw** [comment]
If using the AutocheckingRecheckDriver(RemoteWebDriver, RecheckWebOptions) this makes the test to die producing a dump file. Not a warning any more.

**Comment by diba1013** [comment]
This should be fixed in recheck-1.6.0 for which we have a new release available. If you find any issues regarding this problem, please reopen this ticket.

Please us the following versions:
* [recheck-web-1.6.0](https://github.com/retest/recheck-web/releases/tag/v1.6.0) (which includes [recheck-1.6.0](https://github.com/retest/recheck/releases/tag/v1.6.0)) with either
* [recheck.cli-1.6.0](https://github.com/retest/recheck.cli/releases/tag/v1.6.0) (Open Source) or
* [review-1.6.0](https://assets.retest.org/releases/review/review-1.6.0.zip) or [review-1.6.0-win]( https://assets.retest.org/releases/review/review-1.6.0-win.zip) (Paid)

Edit: Updated to most up to date releases.
